bots/machine/machine_core/cli.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
```

Comment:
I generally like the split into ssh/machine/virtual/cli. But why does that need to go into yet another subdirectory? just keeping the python module dir as bots/machine/ should be fine? There's nothing else on top of "machine_core".

Reply:
@martinpitt that was also my first idea, but I found that this is problematic to rewrite it like that. 

``machine`` is more less fake module and works well because everything is in one file, so no troubles with imports.
In case I've splitted it into more files, it happened to me, that I have to use syntax like ``from .something import anotherthing`` what is still fine. but trouble is that in more places there is used ``import testvm`` what is bad from module perspective (raises error), should be ``from machine import testvm`` (eg. in ``testlib``) what causes that
 * I have to rewrite lots of files and solve these issues in tests. eg. maybe bad ``import parent`` what imports machine dir, not bots dir as path to machine module
 * or second solution is to do like I did that, and do there this subdir as module, and remove this ``machine`` module, because ``testvm.py`` is still posibble to use. and this wraps everything to be backward compatible.